lib/gstreamer/va/decoder.py

```diff
@@ -21,5 +21,7 @@ class DecoderTest(BaseDecoderTest):
 
   def before(self):
     super().before()
-    # TODO: need gst-va to fix target render device
+    renderDevice = get_media().render_device.split('/')[-1]
+    if renderDevice != 'renderD128':
+      self.gstdecoder = self.gstdecoder.replace('va', f'va{renderDevice}')
```

Comment:
It would be better to override the gstdecoder property in the Decoder class for gst-va